lerna.json

```
@@ -2,7 +2,8 @@
   "packages": [
     "packages/*",
     "packages/google/*",
-    "packages/adobe/*"
+    "packages/adobe/*",
+    "packages/quantum-metric/*"
```

This is wrong actually. `google` and `adobe` globs are necessary because their packages resided within subdirectories. The _quantum-metric_ package will be covered by _packages/*_.
```suggestion
    "packages/adobe/*"
```